Dockerfile

```diff
@@ -226,10 +244,12 @@ RUN mkdir -p /opt/collectd-python &&\
 COPY python/ /opt/sfxpython/
 RUN cd /opt/sfxpython && pip install .
 
+RUN pip list
+
 # Delete all compiled python to save space
-RUN find /usr/lib/python2.7 /usr/local/lib/python2.7/dist-packages -name "*.pyc" | xargs rm
+RUN find $PYTHONHOME/lib/python2.7 -name "*.pyc" -o -name "*.pyo" | xargs rm
```

Comment:
I think this was an existing but, but this should delete pyc and compile headers from `/opt/python` for it to have any effect.

Reply:
Changed `$PYTHONHOME/lib/python2.7` to just `$PYTHONHOME`.